src/lib.rs

```diff
@@ -118,6 +122,15 @@ impl<T: FileLoader> VfsInternal<T> {
         }
         result
     }
+
+    fn get_line(&self, path: &Path, line: usize) -> Option<String> {
+        let mut files = self.files.lock().unwrap();
+        if !files.contains_key(path) {
+            let file = T::read(path).unwrap();
+            files.insert(path.to_path_buf(), file);
+        }
+        files.get(path).map(|f| f.get_line(line).to_string())
```

Comment:
I don't know without investigating, but it seems like something that could cause confusion and possibly a problem in the very near future